lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -16,10 +16,14 @@ import {
 } from '../config/ClientConfiguration';
 import { CryptoProvider } from '../crypto/CryptoProvider';
 import { NodeStorage } from '../cache/NodeStorage';
+import { NodeCacheManager } from '../cache/NodeCacheManager';
 
 export abstract class ClientApplication {
     // Input configuration by developer/user
     protected config: ClientConfiguration;
+    protected nodeStorage: NodeStorage;
```

Comment:
We should consider renaming these to get rid of "Node". Maybe cacheStorage would be more precise.

Reply:
Done